modular_skyrat/modules/black_mesa/code/xen_crystal.dm

```diff
@@ -43,7 +43,9 @@
 
 /obj/item/grenade/xen_crystal
 	name = "xen crystal"
-	desc = "A crystal with anomalous properties."
+	desc = "A crystal with anomalous properties. Perhaps its powers could be used to weaken the link between worlds. \
```

Comment:
could this be on examine() instead? Putting html in desc is blegh

Reply:
I guess I can try.